src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -463,6 +475,22 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
 
   const walletSupportsBuyAssetChain = useWalletSupportsChain(buyAsset.chainId, wallet)
 
+  const nativeAssetBridgeWarning: string | [string, InterpolationOptions] | undefined =
+    useMemo(() => {
+      if (!buyAssetFeeAsset) return
+      // TODO(gomes): Bring me in for all bridges?
+      const isBridge = (activeQuote as ArbitrumBridgeTradeQuote)?.direction === 'deposit'
```

Comment:
Only Arb deposits should show this warning for now

Reply:
Then we might want to call it `isArbitrumBridge`

Also the name is a lie for now, it should be either `isBridgeDeposit` or `isArbitrumBridgeDeposit`

Reply:
The idea is to eventually have all bridges show this warning, but agree in the meantime let's probably keep things declarative!
271401cf3